app/scripts/lib/fxa-client.js

```diff
@@ -125,13 +125,33 @@ define(function (require, exports, module) {
         });
     },
 
+    confirmSignInResend: function (relier, sessionToken, options) {
```

Comment:
I'm going to make the call that account.js should be the lowest level abstraction we should update. Since we are re-using existing auth-server endpoints, account.js should call the existing fxa-client methods. We can revisit in the future if needed.